src/shared/constants.h

```diff
@@ -167,6 +167,6 @@ static const char ARDUINO_VENDOR_ID[ARDUINO_ID_LENGTH]  = "2341";
 static const char ARDUINO_PRODUCT_ID[ARDUINO_ID_LENGTH] = "0043";
 
 // Number of times the control loop should tick per trajectory element
-static const unsigned CONTROL_LOOP_HZ = 60u;
+static const unsigned CONTROL_LOOP_HZ = 300u;
```

Comment:
nit: I think this name is from a long time ago. We can probably update it to be a bit more relevant to now
```suggestion
static const unsigned THUNDERLOOP_LOOP_HZ = 300u;
```
or
```suggestion
static const unsigned THUNDERLOOP_HZ = 300u;
```

Reply:
resolved